src/features/import-assertions.md

```diff
@@ -0,0 +1,76 @@
+---
+title: 'Import Assertions'
+author: 'Dan Clark ([@dandclark1](https://twitter.com/dandclark1))'
+avatars:
+  - 'dan-clark'
+date: 2021-06-10
```

Comment:
(TODO: update before merging as this should be publishing date)

Reply:
I bumped this to the 15, on the assumption that this will be published tomorrow.